style.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  width: 320px;
+  height: 4910px;
+  background: #fff;
+}
```

Comment:
- [ ] Please make sure that the design is the same as the figma template as much as possible. it seems that spacing need to be fixed and the design should much the template. and also please make sure that the image under the text is runddering as we see in your design its missing. kindly fix it.
### your design👇
![paddingwright](https://user-images.githubusercontent.com/66992901/198532984-1d814915-da0b-4fcf-a06b-67e26fffa5b2.PNG)
### figma design👇
![figmapadding](https://user-images.githubusercontent.com/66992901/198533297-2b0bfbb1-b5dc-4ec7-81e3-ac21d07e1ef7.PNG)